demisto_sdk/commands/validate/validators/BC_validators/BC102_is_context_path_changed.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from itertools import chain
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration]
+
+
+def is_context_path_changed(content_item: Integration) -> bool:
```

Comment:
can put it inside the class.

Reply:
@dorschw asked to move it outside the class